dashboard/src/components/EncryptionKey.js

```diff
@@ -104,7 +104,9 @@ const EncryptionKey = ({ isMain }) => {
       const encryptedPersons = await Promise.all(persons.map(preparePersonForEncryption).map(encryptItem));
       const encryptedGroups = await Promise.all(groups.map(prepareGroupForEncryption).map(encryptItem));
 
-      const encryptedActions = await Promise.all(actions.map(prepareActionForEncryption).map(encryptItem));
+      const encryptedActions = await Promise.all(
+        actions.map((action) => prepareActionForEncryption({ ...action, user: action.user || user._id })).map(encryptItem)
+      );
```

Comment:
c'est géré dans la PR https://github.com/SocialGouv/mano/pull/1310

```suggestion
       const encryptedActions = await Promise.all(actions.map(prepareActionForEncryption).map(encryptItem));
```